### DISCUSSION TITLE
Feature Request: Enable Invites with `AUTH_DISABLE_SIGNUP` (or/and GitHub OAuth for a given organization)

### DISCUSSION THREAD

**Discussion by shner-elmo** [langfuse/langfuse#4154] [Ideas]
**Feature Request:** 

1. **Enable invites while AUTH_DISABLE_SIGNUP is active** – Allow admins to invite users even when the `AUTH_DISABLE_SIGNUP` setting is enabled, keeping unwanted sign-ups blocked while still enabling controlled access for invited users.
  
2. **Restrict GitHub OAuth sign-ups to members of a specific GitHub organization** – Limit sign-ups to only those GitHub accounts that are members of the organization where the OAuth app is configured, adding an extra layer of access control based on organization affiliation.

**Comment by marcklingen** [comment]
makes sense, thank you for sharing!

**Comment by pzehle** [comment]
So this means that if signups are disabled, I can't invite my co-workers into my instance?

**Reply by marcklingen** [reply]
that's correct, this will block all new signups

**Comment by bobir01** [comment]
@marcklingen anything new on this ?
thanks!

**Comment by loginkeithhe** [comment]
By the way, it seems that this is still the case. After registration is prohibited, invited users can't register and log in.

**Comment by dorabra** [comment]
I have the same issue.
